oak_containers/proto/interfaces.proto

```diff
@@ -38,6 +38,11 @@ message SendAttestationEvidenceRequest {
   oak.session.v1.AttestationEvidence evidence = 1;
 }
 
+message NotifyAppReadyRequest {
+  // The port on which the app is listening.
+  int32 listening_port = 1;
```

Comment:
Shouldn't the app always listen on the same hardcoded port (e.g. 8080)? e.g. that's how Cloud Run works: https://cloud.google.com/run/docs/container-contract#port

Only in the guest of course -- on the host we do need to pick a different port.

Reply:
Good point. I sent the port in case we want to use it, but all the code ignores it so removed again.